File: app-flutter/lib/pages/flutter/ui/title/AboutListTile.dart
import 'package:flutter_web/material.dart';
// ======================================================================================================== 全局配置
import 'package:flutter_web_study.app/config.dart';
// ======================================================================================================== Widget 控件
class AboutListTilePage extends StatefulWidget { // 有状态的控件，控件的状态、文字等属性存在变化的可能
    final String title;
    AboutListTilePage({ Key key, this.title }) : super(key: key);
    @override
    _PageState createState() => _PageState(); // === createState()  => _PageState();
}

class _PageState extends State<AboutListTilePage> {
    // page
    @override
    Widget build(BuildContext context) {
        return Scaffold(
            appBar: Config.appBar(widget.title),
            body: ListView(
                padding:EdgeInsets.all(10.0),
                children: <Widget>[
                    AboutListTile(
                        icon:  CircleAvatar(child: new Text('icon')),
                        child: new Text('flutter'),
                        aboutBoxChildren:<Widget>[
                            Text('aboutBoxChildren'),
                            Text('aboutBoxChildren'),
                            Text('aboutBoxChildren'),
                        ],
                        applicationName:'applicationName',
                        applicationIcon: CircleAvatar(backgroundImage: AssetImage(Config.staticUrl) ),
                        applicationVersion:'applicationVersion',
                        applicationLegalese: "applicationLegalese",
                    ),
                    AboutListTile(
                        icon:  CircleAvatar(child: new Text('2')),
                    )
                ],
            )
        );
    }
}

File: app-react/src/pages/flutter/ui/title/AboutListTile.js
import React from 'react'
// ===================================================================== public component
import ListComponent from '@cpt/list.component'
// ===================================================================== component
const data = {
	title:'关于我们的列表标题',
	keyword:'AboutListTile( )',
	type:'AboutListTile',
	url:'AboutListTilePage',
	data:[
		{ deviceTitle:'列表设置', keyword:'icon', type:'Widget', pro:'列表左侧 icon' },
		{ keyword:'child', type:'widget', pro:'列表标题' },
		{ deviceTitle:'弹窗设置', keyword:'applicationIcon', type:'Widget', pro:'图标' },
		{ keyword:'applicationName', type:'String', pro:'主标题' },
		{ keyword:'applicationVersion', type:'String', pro:'副标题' },
		{ keyword:'applicationLegalese', type:'String', pro:'次标题' },
		{ keyword:'aboutBoxChildren', type:'List<Widget>', pro:'副图片', code:[ {path:'flutter/public/widget-children'} ] },
	]
}
export default class Index extends React.Component{ render(){ return <ListComponent data={data} {...this.props}/> } }